### ISSUE TITLE
JRE Load Error

### ISSUE THREAD

**Issue by kanusinha** [TurboVNC/turbovnc#285]
Hi, I can't open TurboVNC on my mac because of a "JRE Load Error".
How do I fix it? Thanks!

**Comment by dcommander** [comment]
Do you have Java installed? Which version?

**Comment by dcommander** [comment]
Presuming fixed, since I haven't heard from you.

**Comment by ravanfarpa** [comment]
Hi dcommander,
I have downloaded and installed version 2.2.6 on MacBook pro Intel, BigSur 11.4. 
I cannot run the app and get the JRELoadError window. 
I tried the solutions mentioned in another page such as :
sudo ln -fs /Library/Java/JavaVirtualMachines/jdk-11.jdk /Library/Internet\ Plug-Ins/JavaAppletPlugin.plugin
or 
export JAVA_HOME=`/usr/libexec/java_home -v 11`
launchctl setenv JAVA_HOME $JAVA_HOME

but did not help. 
Would appreciate any suggestions. 
Many thanks
